go.mod

```diff
@@ -5,31 +5,34 @@ go 1.18
 require (
 	github.com/AlecAivazis/survey/v2 v2.3.5
 	github.com/TwiN/go-color v1.1.0
-	github.com/cli/go-gh v0.1.0
+	github.com/cli/go-gh v1.2.1
```

Comment:
This is a major version change. Do our e2e tests cover scenarios to ensure that nothing is broken due to this?